modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/servlet/taglib/ChangeTrackingIndicatorDynamicInclude.java

```diff
@@ -604,6 +604,38 @@ private void _getTimelineData(
 		if ((className != null) && (classPK != 0)) {
 			long classNameId = _portal.getClassNameId(className);
 
+			CTCollection currentCTCollection = null;
+
+			CTPreferences ctPreferences =
+				_ctPreferencesLocalService.fetchCTPreferences(
+					themeDisplay.getCompanyId(), themeDisplay.getUserId());
+
+			if (ctPreferences != null) {
+				currentCTCollection =
+					_ctCollectionLocalService.fetchCTCollection(
+						ctPreferences.getCtCollectionId());
+			}
```

Comment:
Update this method to take in the ctCollection as it used to so that we don't need to re-fetch again. https://github.com/liferay-publications/liferay-portal/pull/1054/commits/072d55b64678625124a394160306cc445caaafeb#diff-d468d6d23259e8a0792ad3306600379df1a0258d5016459c7b38fc0fd24dbb22L613